lhotse/bin/modes/kaldi.py

```diff
@@ -42,7 +42,7 @@ def kaldi():
 @click.option(
     "-d",
     "--use-reco2dur",
```

Comment:
I think with default=True and is_flag=True it is impossible to turn this off. The idiomatic click way would be `@click.option("--use-reco2dur/--compute-durations", default=True, help="...")` and then name the Python function argument `use_reco2dur: bool` (the same as the first option before slash, i.e. the same as right now).